src/main/jbake/content/community/docs/developer-guide/alerts.adoc

```diff
@@ -389,3 +451,420 @@ public class Action {
+| *Optional* |
+_trigger.context["url"]_ +
+ +
+Elasticsearch url can be defined in several ways in the alerter. +
+See <<Elasticsearch Security>>. +
+ +
+In addition to the methods defined, a trigger can overwrite the url which it connects. +
+Url can be a list of valid link:https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/HttpHost.html[org.apache.http.HttpHost] urls. By default it will point to +
+ +
+trigger.context["url"] = "http://localhost:9200" +
+ +
+i.e. a valid url could be
+ +
+trigger.context["url"] = "http://host1:9200,http://host2:9200,http://host3:9200"
+
+|=======================
+
+A complete example of an Elasticsearch trigger:
+
+[source,json]
+----
+    {
+      "trigger":{
+        "id": "trigger-project",
+        "name": "Project Logging Trigger",
+        "description": "Alert on Project Logging (EFK infrastructure)",
+        "severity": "HIGH",
+        "enabled": true,
+        "tags": {
+          "Elasticsearch": "Demo ES instance"
+        },
+        "context": {
+          "timestamp": "@timestamp",
+          "interval": "30s",
+          "index": "project.logging*",
+          "mapping": "type|'Unknown':category,@timestamp:ctime,message:text,hostname:dataId,index:tags"
+        },
+        "actions":[
+          {
+            "actionPlugin": "elasticsearch",
+            "actionId": "write-full-alert"
+          },
+          {
+            "actionPlugin": "elasticsearch",
+            "actionId": "write-partial-alert"
+          },
+          {
+            "actionPlugin": "email",
+            "actionId": "email-to-admins"
+          }
+        ]
+      },
+      "conditions":[
+        {
+          "type": "EVENT",
+          "dataId": "192.168.122.198",
+          "expression": "category == 'response'"
+        }
+      ]
+    }
+----
+
+=== Elasticsearch Action Plugin
+
+It processes Actions writing Event/Alerts into an Elasticsearch system. +
+ +
+The Elasticsearch plugin supports following properties: +
+ +
+[cols="^2,10,^2", options="header"]
+|=======================
+| Property |
+Description |
+Default value
+| *url* |
+See <<Elasticsearch Security>>. +
+ +
+Indicate the Elasticsearch server or servers to connect. +
+|
+_http://localhost:9200_
+| *index* |
+Indicate the index where the Events/Alerts will be written.
+| _alerts_
+| *type* |
+Define the type under the index where the Events/Alerts will be written.
+| _hawkular_
+| *transform* |
+Define an optional transformation expression based on JOLT Shiftr format to convert an Event/Alert into a custom JSON format. +
+ +
+i.e. +
+{"tenantId":"tenant","ctime":"timestamp","dataId":"dataId","context":"context"} +
+ +
+link:https://github.com/bazaarvoice/jolt/blob/master/jolt-core/src/main/java/com/bazaarvoice/jolt/Shiftr.java[JOLT Shiftr JavaDoc] +
+link:http://jolt-demo.appspot.com/[JOLT Online tool]
+| -
+| *user* |
+See <<Elasticsearch Security>>. +
+ +
+Username for Basic credential authentication.
+| -
+| *pass* |
```

Comment:
I brought this from mail.smtp.pass which is used in the Email plugin, which I tried to see how other products used.
i.e.
https://www.playframework.com/documentation/1.2.4/configuration#mail